gen3-cli/cot/backend/create/blueprint/__init__.py

```diff
@@ -0,0 +1,21 @@
+from cot.backend.common import runner
+
+
+def run(
+    generation_input_source=None,
+    output_dir=None,
+    generation_provider=None,
+    generation_framework=None,
+    generation_testcase=None,
+    generation_scenarios=None,
+    _is_cli=False
+):
+    options = {
+        '-i': generation_input_source,
+        '-o': output_dir,
+        '-p': generation_provider,
+        '-f': generation_framework,
+        '-t': generation_testcase,
+        '-s': generation_scenarios
+    }
+    runner.run('createBlueprint.sh', [], options, _is_cli)
```

Comment:
please configure your editor to add newlines to the file end; some standard requires it

Reply:
Usually, it adds new lines at the end of files, maybe this is a bug.